README.md.mustache

```diff
@@ -26,8 +29,13 @@
 [travis-link]: https://travis-ci.com/{{ organization }}/{{ shortname }}/builds
 {{/ travis }}
 {{# action }}
-[action-shield]: https://github.com/{{ organization }}/{{ shortname }}/workflows/CI/badge.svg?branch={{branch}}{{^branch}}master{{/branch}}
-[action-link]: https://github.com/{{ organization }}/{{ shortname }}/actions?query=workflow%3ACI
+[docker-action-shield]: https://github.com/{{ organization }}/{{ shortname }}/workflows/docker-action.yml/badge.svg?branch={{branch}}{{^branch}}master{{/branch}}
```

Comment:
I made a test and this yields a 404 error.

Write instead:

```suggestion
[docker-action-shield]: https://github.com/{{ organization }}/{{ shortname }}/workflows/Docker-based%20CI/badge.svg?branch={{branch}}{{^branch}}master{{/branch}}
```

Reply:
According to the [GitHub doc](https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge) what I wrote is supposed to work as well. But it turns out that your solution is superior because providing the title makes it possible for GitHub to generate a GitHub badge even before the workflow exists (better for previews).